Updates to incorporate Ines Robles's review

mingpeiwk opened the pull request:
Give an example about "In some use cases it may be sufficient to identify only the class of the device".

See Issue #250

mingpeiwk committed:
Updates to incorporate Ines Robles's review

Give an example about "In some use cases it may be sufficient to identify only the class of the device".

dthaler commented on draft-ietf-teep-architecture.md:
```diff
@@ -1150,7 +1151,8 @@ The following information is required for TEEP attestation:
     TAs, and providing subscriptions to services, and locating device-specific keying material to
     communicate with or authenticate the device. In some use cases it may be sufficient to identify 
-    only the class of the device. The security and privacy requirements regarding device identification 
+    only the class of the device, for example, a DAA Issuer's group public key ID when the attestation uses DAA, 
+    see {{I-D.ietf-rats-daa}}. The security and privacy requirements regarding device identification 
```
Approving though I think originally (based on the OTrP text) this was referring to what [I-D.ietf-rats-eat] section 4.2.4 
 calls hardware-model.   Not sure if we should add "model or"?

```suggestion
    only the model or class of the device, for example, a DAA Issuer's group public key ID when the attestation uses DAA, 
    see {{I-D.ietf-rats-daa}}. The security and privacy requirements regarding device identification 
```

mingpeiwk commented:
Will change to add this for better match. Model can work better than class.

Ming

Sent from my iPhone

On Oct 19, 2022, at 7:36 AM, Dave Thaler ***@***.***> wrote:

﻿

***@***.**** approved this pull request.
------------------------------

In draft-ietf-teep-architecture.md
<https://github.com/ietf-teep/architecture/pull/260#discussion_r999549747>:

> +    only the class of the device, for example, a DAA Issuer's group public key ID when the attestation uses DAA,
+    see {{I-D.ietf-rats-daa}}. The security and privacy requirements
regarding device identification

Approving though I think originally (based on the OTrP text) this was
referring to what [I-D.ietf-rats-eat] section 4.2.4
calls hardware-model. Not sure if we should add "model or"?
⬇️ Suggested change

-    only the class of the device, for example, a DAA Issuer's group
public key ID when the attestation uses DAA,
-    see {{I-D.ietf-rats-daa}}. The security and privacy requirements
regarding device identification
+    only the model or class of the device, for example, a DAA
Issuer's group public key ID when the attestation uses DAA,
+    see {{I-D.ietf-rats-daa}}. The security and privacy requirements
regarding device identification

—
Reply to this email directly, view it on GitHub
<https://github.com/ietf-teep/architecture/pull/260#pullrequestreview-1147717065>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AGMR6F4GM3IFXZYUIVXNFDDWEABNTANCNFSM6AAAAAARI4GJHI>
.
You are receiving this because you authored the thread.Message ID:
***@***.***>

-- 
This electronic communication and the information and any files transmitted 
with it, or attached to it, are confidential and are intended solely for 
the use of the individual or entity to whom it is addressed and may contain 
information that is confidential, legally privileged, protected by privacy 
laws, or otherwise restricted from disclosure to anyone else. If you are 
not the intended recipient or the person responsible for delivering the 
e-mail to the intended recipient, you are hereby notified that any use, 
copying, distributing, dissemination, forwarding, printing, or copying of 
this e-mail is strictly prohibited. If you received this e-mail in error, 
please return the e-mail to the sender, delete it from your computer, and 
destroy any printed copy of it.

mingpeiwk committed:
Update draft-ietf-teep-architecture.md

Co-authored-by: Dave Thaler <dthaler@microsoft.com>

mingpeiwk committed:
Add an example of model for device

mingpeiwk committed:
Remove section number in EAT ID in case it changes

Use a more specific term "hwmodel" in the other EAT ID without using section numbers to avoid future change needs when the other ID change sections.
